workitem/enum_type_whitebox_test.go

```diff
@@ -22,6 +22,67 @@ func TestEnumTypeContains(t *testing.T) {
 	assert.False(t, contains(haystack, needle))
 }
 
+func TestEnumTypeContainsAll(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
+	haystack := []interface{}{1, 2, 3, 4}
+
+	// Check single value
+	needles := []interface{}{1}
```

Comment:
@michaelkleinhenz please use subtests instead of a comment, then we can easily spot if one of the use-cases fails. It requires just one more line for the closing bracket and parenthesis.

Reply:
Fixed. 5e88ed5.